kimchi/src/circuits/polynomials/range_check/circuitgates.rs

```diff
@@ -137,77 +133,79 @@ where
 
     // Constraints for RangeCheck0
     //   * Operates on Curr row
-    //   * Range constrain all sublimbs except p4 and p5 (barring plookup constraints, which are done elsewhere)
-    //   * Constrain that combining all sublimbs equals the limb stored in column 0
+    //   * Range constrain all limbs except vp4 and vp5 (barring plookup constraints, which are done elsewhere)
+    //   * Constrain that combining all limbs equals the limb stored in column 0
     fn constraints() -> Vec<E<F>> {
-        // 1) Apply range constraints on sublimbs
+        // 1) Apply range constraints on limbs
         // Columns 1-4 are 12-bit plookup range constraints (these are specified elsewhere)
         // Create 8 2-bit chunk range constraints
         let mut constraints = (7..COLUMNS)
             .map(|i| crumb(&witness_curr(i)))
             .collect::<Vec<E<F>>>();
 
-        // 2) Constrain that the combined sublimbs equals the limb stored in w(0) where
-        //    limb = lp0 lp1 lp2 lp3 lp4 lp5 lc0 lc1 lc2 lc3 lc4 lc5 lc6 lc7
+        // 2) Constrain that the combined limbs equals the limb stored in w(0) where
+        //    v = vp0 vp1 vp2 vp3 vp4 vp5 vc0 vc1 vc2 vc3 vc4 vc5 vc6 vc7
         //    in big-endian byte order.
         //
         //          Columns
         //          0      1    2    3    4    5    6    7    8    9    10   11   12   13   14
-        //    Curr  limb   lp0  lp1  lp2  lp3  lp4  lp5  lc0  lc1  lc2  lc3  lc4  lc5  lc6  lc7  <- LSB
+        //    Curr  v      vp0  vp1  vp2  vp3  vp4  vp5  vc0  vc1  vc2  vc3  vc4  vc5  vc6  vc7  <- LSB
         //
-        // Check limb =  lp0*2^0 + lp1*2^{12}  + ... + p5*2^{60}   + lc0*2^{72}  + lc1*2^{74}  + ... + lc7*2^{86}
+        // Check v    =  vp0*2^0 + vp1*2^{12}  + ... + p5*2^{60}   + vc0*2^{72}  + vc1*2^{74}  + ... + vc7*2^{86}
         //       w(0) = w(1)*2^0 + w(2)*2^{12} + ... + w(6)*2^{60} + w(7)*2^{72} + w(8)*2^{74} + ... + w(14)*2^{86}
         //            = \sum i \in [1,7] 2^{12*(i - 1)}*w(i) + \sum i \in [8,14] 2^{2*(i - 7) + 6*12}*w(i)
```

Comment:
I think the last formula is imprecise with the open/closed parenthesis.
Either 
`\sum i \in [1,6] 2^{12*(i - 1)} * w(i) + \sum i \in [7, 14] 2^{2*(i - 7) + 6*12 } * w(i) `
or
`\sum i \in [1,7) 2^{12*(i - 1)} * w(i) + \sum i \in (8, 14] 2^{2*(i - 7) + 6*12 } * w(i) `

Reply:
You're right - the comment matches the old code.  Thanks.  Will update in next PR.